frontend/pages/SoftwarePage/SoftwareVulnerabilityDetailsPage/SoftwareVulnerabilityDetailsPage.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useContext } from "react";
+import { useQuery } from "react-query";
+import { RouteComponentProps } from "react-router";
+
+import { AxiosError } from "axios";
+
+import { AppContext } from "context/app";
+
+import { IVulnerability } from "interfaces/vulnerability";
+
+import softwareVulnAPI, {
+  IVulnerabilityResponse,
+} from "services/entities/vulnerabilities";
+
+import Spinner from "components/Spinner";
+import DataError from "components/DataError";
+import Fleet404 from "pages/errors/Fleet404";
+import MainContent from "components/MainContent";
+
+import SoftwareVulnSummary from "./SoftwareVulnSummary/SoftwareVulnSummary";
+import SoftwareVulnOSVersions from "./SoftwareVulnOSVersions";
+import SoftwareVulnSoftwareVersions from "./SoftwareVulnSoftwareVersions";
+
+const baseClass = "software-vulnerability-details-page";
+
+interface ISoftwareVulnerabilityDetailsRouteParams {
+  cve: string;
+}
+
+type ISoftwareVulnerabilityDetailsPageProps = RouteComponentProps<
+  undefined,
+  ISoftwareVulnerabilityDetailsRouteParams
+>;
+
+const SoftwareVulnerabilityDetailsPage = ({
+  router,
+  routeParams,
+}: ISoftwareVulnerabilityDetailsPageProps) => {
+  const { isPremiumTier } = useContext(AppContext);
+
+  const cve = routeParams.cve;
+
+  const {
+    data: vuln,
+    isLoading: isVulnLoading,
+    isError: isVulnError,
+    error: vulnError,
+  } = useQuery<IVulnerabilityResponse, AxiosError, IVulnerability>(
+    ["softwareVulnByCVE", cve],
+    () => softwareVulnAPI.getVulnerability(cve),
+    {
+      select: (data) => data.vulnerability,
```

Comment:
maybe add the frontend sort here and leave a comment that the table is not sortable but it is sorted by host count here